packages/react-server-cli/src/coreCssMiddleware.js

```diff
@@ -0,0 +1,13 @@
+
+module.exports = function(pathToStatic) {
```

Comment:
Ugh, I think I was wrong about being able to get rid of `window.__reactServerBase`. It's used when the JS is compiled using `compileOnly` and the server is launched with `jsUrl` to tell the server code where the compiled client code lives.

In that case, the compiled client JS has `null` for `pathToStatic`, whereas the server code has the real runtime `pathToStatic`. `window.__reactServerBase` is a sneaky way to pass the path from the server to the client without having to recompile the client routes file.